content/docs/ref/ci.md

```diff
@@ -0,0 +1,22 @@
+# Command Reference: `ci`
+
+```bash
+cml ci [options]
+```
+
+Fixes CI setups.
```

Comment:
A bit laconic, perhaps? Maybe it's worth mentioning that this tool is meant to leave Git repositories in a consistent state on every platform. If it “fixed CI setups” as advertised, we would be the number one project in the world.

Reply:
```suggestion
Miscellaneous CI/CD setup/preparation.
```

Reply:
> _setup/preparation_

Would “normalization” be too pretentious/vague/overloaded? If not, it could be a better alternative.

Reply:
> _Miscellaneous_

My sixth sense reacts against “miscellaneous” and “et cetera” in tool descriptions, but...

Reply:
Configures a Git repository for use on CI workflows?

Running out of imagination. It's probably better to leave it as–is.

Reply:
how about the updated suggestion?